class ExceptionSize(Exception):
    pass

class ValidacaoError(Exception):
    pass
